meshtastic/__main__.py

```diff
@@ -1423,6 +1429,18 @@ def initParser():
         action="append",
     )
 
+    group.add_argument(
+        "--channel-fetch-retries",
+        help=("Attempt to retrieve channel settings for --ch-set this many times before giving up."),
+        default=3,
+    )
+
+    group.add_argument(
+        "--timeout",
+        help="How long to wait for replies",
+        default=300
+    )
+
```

Comment:
I think we can make this code a little nicer by adding `type=int` to both of these arguments, at which point it also shouldn't be necessary to convert them to ints later.